specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -7046,7 +7046,7 @@
           "description": "The type of the label",
           "enum": [
             "User",
-            "System"
+            "AutoAssigned"
```

Comment:
I see your PR updates existing preview API version 2019-01-01-preview which is older than 12 months. Preview APIs have a life-span of not more than 12 months, after which they must be retired. As you are also updating one of the existing enum values, I would recommend to introduce a new preview API version.

Please refer documentation on Preview APIs here https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html#preview-apis.

Reply:
@RamyasreeChakka I was unaware of the 12 month limit. I'll start the process internally for us to create a new preview API version and once the new version is live and working we'll retire the existing preview version according to the API deprecation procedure. 
Since that process can take a while and in the meantime customers that want to generate a PowerShell client from our swagger (for both preview and stable versions) are unable to do so, I still think there's value in unblocking them in the meantime by performing this change. What do you think?

Reply:
Sounds good. Signed off.